### DISCUSSION TITLE
Netbox add one device can use to position

### DISCUSSION THREAD

**Discussion by channysreng** [netbox-community/netbox#16658] [General]
Hello all, 

I have one question on netbox, I have one server DELL current is rack it used U 10-11 but in netbox I try to add it not possible to use two U10-11 it allow to use only one U. Who know this and how to fix?

Thanks

**Comment by a084ed22** [comment]
Your device template definition should specify your device is 2U.

**Comment by JessicaRigol** [comment]
You should edit the device in device types.
![image](https://github.com/netbox-community/netbox/assets/112140945/3e2c95e9-ff40-4760-ab7b-c53bdeaa4a86)
